### ISSUE TITLE
Upgrade Weblate

### ISSUE THREAD

**Issue by BudsieBuds** [opensourcepos/opensourcepos#3203]
Please upgrade Weblate to the latest version for everyone to enjoy all sparkling new features, speed improvements and fixes.

Current version: 4.3 (October 15th 2020)
Latest version at this time: 4.6.2 (May 8th 2021)
Upcoming version: 4.7 (due June 15th 2021)

Upgrade documentation here: https://docs.weblate.org/en/latest/admin/upgrade.html

Thanks!

**Comment by jekkos** [comment]
I must say that this weblate maintenance has been quite a nightmare, but that might also be because of the way I had it setup. Every upgrade I ended up reinitializing the database as there was always some issue with the django db migrations. 
This bad experience might have been the result from the fact that I was running the bleeding edge (dev version).

Last time I took the decision to switch to a version managed by pip so hopefully the upgrade experience will be a bit smoother this time.
